dns.go

```diff
@@ -46,6 +46,41 @@ func (api *API) AddSubdomain(domain string, subdomain string) (*Status, error) {
 	}, nil
 }
 
+// AddZoneRecord - Create zone record
+func (api *API) AddZoneRecord(domain string, subdomain string, record *Record) error {
+	var result string
+	args := []interface{}{
+		api.Username,
+		api.Password,
+		api.CustomerNumber,
+		domain,
+		subdomain,
+		record,
+	}
+	if err := api.XMLRPCClient().Call("addZoneRecord", args, &result); err != nil || result != "OK" {
+		return err
+	}
+
+	results, err := api.GetZoneRecords(domain, subdomain)
+	if err != nil {
+		return err
+	}
+
+	for _, element := range results {
+		// Exclude ID before equality check
+		id := element.ID
+		element.ID = 0
+
+		// Compare by value
+		if *&element == *record {
```

Comment:
is this bad practice? @akarl818

Reply:
Looks fine. Although wouldn't `element == *record` be the same thing?

Reply:
That looks a lot simpler! Thanks for the suggestion